recipes-kernel/linux/files/extra-kernel-config.cfg

```diff
@@ -1,3 +0,0 @@
-CONFIG_OVERLAY_FS=y
```

Comment:
are these configs pulled in by some other means?  I think we still need these for normal wigwag upgrade scenarios, right?

Reply:
@x85446 how are we enabling overlayfs in 1.0?